cni/pkg/repair/repair.go

```diff
@@ -44,19 +45,26 @@ type Filters struct {
 	LabelSelectors                  string `json:"label_selectors"`
 }
 
+type Metrics struct {
```

Comment:
curious: why intro this struct?  Is this just for testing?

it seems like an exported `repair.PodsRepaired` metric would work just as well?  As it is not dimensioned, it doesn't seem like it needs to be passed around like a context.

Reply:
It was the only way I could think of to set it up so I could mock the metrics in tests. If there's a more idiomatic way to do it, it can be removed.